src/components/Editor.jsx

```diff
@@ -1,56 +1,61 @@
-import React, { useState } from "react";
+import { useState, useCallback, useEffect } from "react";
 import { FormLayout, TextField, Button } from "@shopify/polaris";
 import "./Editor.css";
 
-function Editor({ jsonCode, onJsonChange, updatedJson, error, clearJsonCode }) {
```

Comment:
Make sure you pull the latest upstream repository code to your branch before you start working and before creating a PR so all new additions in between can be updated in your branch as well. Here an extra feature to clear the code had been added which appears to be replaced by extra additions so that's not ideal. Can you pull all changes from upstream into your repository and then add your changes on top of it?

Reply:
ok i will resolve all the conflicts